sklearn/inspection/tests/test_plot_partial_dependence.py

```diff
@@ -399,20 +407,55 @@ def test_plot_partial_dependence_dataframe(pyplot, clf_boston, boston):
      (dummy_classification_data,
      {'features': ['foobar'], 'feature_names': ['abcd', 'def']},
       'Feature foobar not in feature_names'),
-     (dummy_classification_data, {'features': [(1, 2, 3)]},
-      'Each entry in features must be either an int, '),
-     (dummy_classification_data, {'features': [1, {}]},
-      'Each entry in features must be either an int, '),
-     (dummy_classification_data, {'features': [tuple()]},
-      'Each entry in features must be either an int, '),
      (dummy_classification_data,
       {'features': [123], 'feature_names': ['blahblah']},
       'All entries of features must be less than '),
      (dummy_classification_data,
       {'features': [0, 1, 2], 'feature_names': ['a', 'b', 'a']},
       'feature_names should not contain duplicates')]
 )
-def test_plot_partial_dependence_error(pyplot, data, params, err_msg):
+def test_plot_common_error(pyplot, data, params, err_msg):
+    X, y = data
+    estimator = LinearRegression().fit(X, y)
+
+    with pytest.raises(ValueError, match=err_msg):
+        plot_partial_dependence(estimator, X, **params)
+    with pytest.raises(ValueError, match=err_msg):
```

Comment:
a general remark for the test. If we expect the same behaviour for some part, let's just parametrize the test with the function to try. It will be more elegant with less repetitive code.

Reply:
Wouldn't this mean duplicating the whole parameter set (https://github.com/scikit-learn/scikit-learn/pull/16164/files/0fa426c083af1e7adeee72dc1fb134d756ce445d#diff-1206a47695776d1666e6e6873502b8e5R398-R415)?